infos.json

```diff
@@ -375,6 +375,22 @@
         "preferences": {
             "key_misc_rounded_corners": "30"
         }
-    }
+    },
+    {
+        "match_property": "vendor_fp",
+        "match_value": "Realme/rmx2020*",
```

Comment:
Hum you're sure? Oppo devices have nasty habits of not having a vendor fingerprint

Reply:
Yes this matches